modules/ossm-configuring-the-threescale-wasm-auth-module.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-configuring-the-threescale-wasm-auth-module_{context}"]
+= Configuring the threescale-wasm-auth module
+
+Cluster administrators on {product-title} can configure the `threescale-wasm-auth` module to authorize HTTP requests to 3scale API Management through an application binary interface (ABI). The ABI defines the interaction between host and the module, exposing the hosts services, and allows you to use the module to process proxy requests.
+
+[id="the-service-mesh-extension_{context}"]
+== The Service Mesh extension
+
+{ProductShortName} provides a xref:../../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[custom resource definition] to specify and apply Proxy-WASM extensions to sidecar proxies, known as xref:../../service_mesh/v2x/ossm-extensions.adoc[`ServiceMeshExtension`]. {ProductShortName} applies this custom resource to the set of workloads that require HTTP API management with 3scale.
+
+[NOTE]
+====
+Configuring the the WebAssembly extension is currently a manual process. Support for fetching the configuration for services from the 3scale system will be available in a future release.
+====
+
+.Prerequisites
+
+* Identify a Kubernetes workload and namespace on your {ProductShortName} deployment that you will apply this module.
+* You must have a 3scale tenant account. See link:https://www.3scale.net/signup[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.11/html-single/installing_3scale/index#install-threescale-on-openshift-guide[3scale 2.11 On-Premises] with a matching service and relevant applications and metrics defined.
+* If you apply the module to the `productpage` microservice in the `bookinfo` namespace, see the xref:ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo sample application].
```

Comment:
Looks like this might need a full path? Is it this assembly:

```
xref:../../service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo sample application]
```